src/main/scala/viper/gobra/frontend/info/implementation/resolution/AmbiguityResolution.scala

```diff
@@ -98,6 +98,8 @@ trait AmbiguityResolution { this: TypeInfoImpl =>
       case Right(_) => None // unknown pattern
     }
 
+    case b: PBlankIdentifier => Some(ap.BlankIdentifier(b))
```

Comment:
why do we need ambiguity resolution? Isn't it straight forward what it is?

Reply:
I don't think so. This case allows us to retrieve the exact location where a `PBlankIdentifier` occurs. The location is important in order to find the type `T` of the expression assigned to the blank identifier. Furthermore, T must be known during desugaring because blank identifiers are desugard into fresh variables of type `T`

in order to create a fresh variable in the place of the identifier of the same type. In order to compute the